### PR TITLE
Enable to get column position on error

### PR DESCRIPTION
**Pull request by steiley**
I implemented for "Advanced syntax check" in TODOs.
#19

I'm not sure if the name of the function to get a column position is good.

### CODE REVIEW

File: lib/language_server/linter/ruby_wc.rb [hunk 1]
```diff
@@ -27,7 +28,7 @@ def initialize(source)
 
       def call
         error_message.scan(/.+:(\d+):\s*(.+?)[,:]\s(.+)/).map do |line_num, type,  message|
-          Error.new(line_num: line_num.to_i - 1, message: message, type: type)
+          Error.new(line_num: line_num.to_i - 1, character: get_character_from_message(error_message), message: message, type: type)
```
Comment (mtsmfm) [inline lib/language_server/linter/ruby_wc.rb]: Can you implement `start` and `end` instead of `character`?

Comment (steiley) [reply lib/language_server/linter/ruby_wc.rb]: Yes.
Implementing this is easy.
But I don't know what to do with the value of end.
Is the end value acceptable at the end of the line?

Comment (mtsmfm) [reply lib/language_server/linter/ruby_wc.rb]: If ruby tells us the position, `start + 1` is `end`.
If doesn't, `start` is 0 and `end` is the number of characters which the line has.

**Comment by mtsmfm** [comment]
Thanks!

**Pull request merged**